extensions/dicom-video/src/viewports/OHIFCornerstoneVideoViewport.tsx

```diff
@@ -30,6 +32,9 @@ function OHIFCornerstoneVideoViewport({ displaySets }) {
         controlsList="nodownload"
         preload="auto"
         className="w-full h-full"
+        crossOrigin={(appConfig.videoViewport && appConfig.videoViewport.crossOrigin) ? appConfig.videoViewport.crossOrigin : undefined}
```

Comment:
Are these new settings in the config? If so, should we add them in the default config just to "show" that they are available? Also should we document these in the extension's README instead? Apologies if I am totally off base with this. :)

Reply:
I've added a config for autoPlay but will leave the cross-origin out as per documentation, adding this tag will change the behavior regardless which option is picked